### DISCUSSION TITLE
Linux on ESP32-S3

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#12279] [Core Development]
I think upython on an esp32 is the best thing since sliced bread, but seen multiple reports lately lately of folks attempting to run linux on esp32s. I can only conclude that they are gluttons for punishment?
![image](https://github.com/micropython/micropython/assets/92007731/2b0fb1a3-1144-4dc1-873e-b00c1eb960b9)

**Comment by mattytrentini** [comment]
At yesterday's PyCon AU sprint day I ran MicroPython on Linux on the ESP32-S3 (an [M5STack CoreS3](https://shop.m5stack.com/products/m5stack-cores3-esp32s3-lotdevelopment-kit)). AMA!

The usefulness of this system is debatable...

**Reply by kjm1102** [reply]
Just chmod 777 at the start of every script Matt, I'm sure it will be fine!
